docs/source/modules/tutorial.rst

```diff
@@ -0,0 +1,653 @@
+        wrappers.
+        :param name: the name of the model to fetch
+        :return: the model instance
+        """
+        assert name == 'my-model'
+
+        # link the custom model to the wrapper object(activations_model above):
+        wrapper = activations_model
+        wrapper.image_size = 224
+        return wrapper
+
+
+    # get_layers method to tell the code what layers to consider. If you are submitting a custom
+    # model, then you will most likley need to change this method's return values.
+    def get_layers(name):
+        """
+        This method returns a list of string layer names to consider per model. The benchmarks maps brain regions to
+        layers and uses this list as a set of possible layers. The lists doesn't have to contain all layers, the less the
+        faster the benchmark process works. Additionally the given layers have to produce an activations vector of at least
+        size 25! The layer names are delivered back to the model instance and have to be resolved in there. For a pytorch
+        model, the layer name are for instance dot concatenated per module, e.g. "features.2".
+        :param name: the name of the model, to return the layers for
+        :return: a list of strings containing all layers, that should be considered as brain area.
+        """
+
+        # quick check to make sure the model is the correct one:
+        assert name == 'my-model'
+
+        # returns the layers you want to consider
+        return  ['conv1', 'relu1', 'relu2']
+
+    # Bibtex Method. For submitting a custom model, you can either put your own Bibtex if your
+    # model has been published, or just return a string like "N/A: Custom Model".
+    def get_bibtex(model_identifier):
+        """
+        A method returning the bibtex reference of the requested model as a string.
+        """
+
+        # from pytorch.py:
+        return "N/A: Custom Model"
+
+    # Main Method: In submitting a custom model, you should not have to mess with this.
+    if __name__ == '__main__':
+        # Use this method to ensure the correctness of the BaseModel implementations.
+        # It executes a mock run of brain-score benchmarks.
+        check_models.check_base_models(__name__)
+
+
+
+
+5. The first is the imports: you will most likely need all of them that
+   the code above has listed. If you try to run the above code in Google Colab
+   (which is basically a Google version of Jupyter Notebooks), it will not
+   run (due to packages not being installed), and is just for visual
+   purposes only; copy and paste the code into your ``models/base_models.py`` file.
+   Next, you see the class definition of the custom model in Pytorch, followed by .
+   preprocessing, the ``PytorchWrapper`` that
+   converts a model into a neuroscience-ready network to run benchmarks on,
+   and  the layers of the network that will be scored.
```

Comment:
"and  the layers of the network that will be scored." -> "and the `ModelCommitment` to convert the activations model into a BrainModel to run on the benchmarks."